roles/network_plugin/multus/templates/multus-daemonset.yml.j2

```diff
@@ -61,6 +61,7 @@ spec:
         - "--cni-bin-dir={{ multus_cni_bin_dir }}"
         - "--multus-conf-file={{ multus_conf_file }}"
         - "--multus-kubeconfig-file-host={{ multus_kubeconfig_file_host }}"
+        - "--namespace-isolation={{ multus_namespace_isolation | default(true) | string | lower }}"
```

Comment:
If already specified in `roles/network_plugin/multus/defaults/main.yml`, there is no need to add `default` here.

Reply:
OK I'll remove it.